aggregator/src/main/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJob.java

```diff
@@ -38,19 +38,22 @@
     @Channel("aggregation")
     Emitter<String> emitter;
 
-    public void processDailyEmail(Instant scheduledFireTime) {
-        List<AggregationCommand> aggregationCommands = processAggregateEmails(scheduledFireTime);
+    public void processDailyEmail() {
+        List<AggregationCommand> aggregationCommands = processAggregateEmails();
+
         List<CompletableFuture<Void>> futures = new ArrayList<>();
         for (AggregationCommand aggregationCommand : aggregationCommands) {
             try {
                 final String payload = objectMapper.writeValueAsString(aggregationCommand);
-                emitter.send(payload);
                 futures.add(emitter.send(payload).toCompletableFuture());
             } catch (JsonProcessingException e) {
                 LOG.warning("Could not transform AggregationCommand to JSON object.");
             }
         }
 
+        final CronJobRun lastCronJobRun = emailAggregationResources.getLastCronJobRun();
+        emailAggregationResources.updateLastCronJobRun(lastCronJobRun.getId(), Instant.now());
```

Comment:
Related to my previous comment:
```suggestion
        emailAggregationResources.updateLastCronJobRun(lastCronJobRun.getId(), now);
```